src/extensions/core/load3d.ts

```diff
@@ -2,63 +2,53 @@
 import { IWidget } from '@comfyorg/litegraph'
 import { IStringWidget } from '@comfyorg/litegraph/dist/types/widgets'
 import PrimeVue from 'primevue/config'
-import { createApp, h, nextTick, render } from 'vue'
+import { createApp, nextTick } from 'vue'
 
 import Load3D from '@/components/load3d/Load3D.vue'
 import Load3DAnimation from '@/components/load3d/Load3DAnimation.vue'
+import { useChainCallback } from '@/composables/functional/useChainCallback'
 import Load3DConfiguration from '@/extensions/core/load3d/Load3DConfiguration'
 import Load3dAnimation from '@/extensions/core/load3d/Load3dAnimation'
 import Load3dUtils from '@/extensions/core/load3d/Load3dUtils'
+import { CustomInputSpec } from '@/schemas/nodeDef/nodeDefSchemaV2'
 import { api } from '@/scripts/api'
 import { app } from '@/scripts/app'
+import { ComponentWidgetImpl } from '@/scripts/domWidget'
 import { useLoad3dService } from '@/services/load3dService'
+import { useDomWidgetStore } from '@/stores/domWidgetStore'
 import { useToastStore } from '@/stores/toastStore'
+import { generateUUID } from '@/utils/formatUtil'
 
 app.registerExtension({
   name: 'Comfy.Load3D',
 
-  getCustomWidgets(app) {
+  getCustomWidgets() {
     return {
-      LOAD_3D(node, inputName) {
-        node.addProperty('Camera Info', '')
-
-        const container = document.createElement('div')
-        container.classList.add('comfy-load-3d')
-
-        /* Hold off for now
-        const mountComponent = () => {
-          const vnode = h(Load3D, {
-            node: node,
-            type: 'Load3D'
-          })
-
-          render(vnode, container)
-        }
-         */
-
-        let controlsApp = createApp(Load3D, {
+      LOAD_3D(node) {
+        const inputSpec: CustomInputSpec = {
+          name: 'image',
           node: node,
```

Comment:
No need to pass node as part of InputSpec, as node is already passed to `ComponentWidgetImpl` constructor.

Reply:
done